src/cli/telemetry/telemetry.ts

```diff
@@ -155,77 +132,79 @@ export const prepareData = async (duration_ms: number, component_count: number =
  * Reads package-lock.json and package.json files in order to cross references the dependencies and devDependencies properties. Pull the current installed version of each package under the @stencil, @ionic, and @capacitor scopes.
  * @returns string[]
  */
-async function getInstalledPackages() {
+async function getInstalledPackages(sys: d.CompilerSystem, config: d.Config): Promise<{ packages: string[] }> {
+  let packages: string[] = [],
+    packageLockJson: any;
```

Comment:
Can we please abstain from the comma operator here, unless there's a good reason to use it? IMO it makes our non-minified code harder to read